contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -771,57 +766,26 @@
       },
 
       /* IMAGE MENU */
-      /**
-       * Initialize elements with image field class with ImageField component
-       */
-      initImageFields({ newOnly = false } = {}) {
-        for (let imageEl of this.imageEls) {
-          if (!newOnly || imageEl.classList.contains(CLASS_IMG_FIELD_NEW)) {
-            const ImageComponent = new ImageFieldClass({
-              propsData: {
-                src: imageEl.getAttribute('src'),
-                alt: imageEl.getAttribute('alt'),
-                width: imageEl.getAttribute('width'),
-                height: imageEl.getAttribute('height'),
-              },
-            });
-            ImageComponent.$mount();
-            ImageComponent.$on('edit', ({ event, component, image }) => {
```

Comment:
These listeners weren't getting created in time, and didn't seem to be getting cleaned up properly either.  It was more straightforward to have the `MarkdownImageField` components just dispatch an event that bubbles up and can be listened to on the editor using a single event listener.